daras_ai_v2/gooey_builder.py

```
@@ -225,3 +192,122 @@ def get_conversation_data_for_saved_run(
+
+@router.post("/__/gooey-builder/send-message", dependencies=[fastapi_login_required])
+def gooey_builder_send_message(request: fastapi.Request, body: GooeyBuilderSendMessage):
+    from daras_ai_v2.workflow_url_input import url_to_runs
+    from functions.gooey_builder_tools import insert_gooey_builder_variables
+
+    workspace = get_current_workspace(request.user, request.session)
+
+    # copy the workflow_url into a new run linked to
+    # builder_sr so the chat widget can navigate the user to a workflow page
+    # that knows which builder iteration produced it
+    workflow_page_cls, workflow_sr, workflow_pr = url_to_runs(body.workflow_url)
```

Ruff (0.15.15)

[warning] Unpacked variable `workflow_page_cls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)